pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,67 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("create", "delete", "deletecollection", "get", "list", "patch", "update", "watch"),
+					Resources: sets.NewString("imagestreamimages", "imagestreamimports", "imagestreammappings", "imagestreams", "imagestreams/secrets", "imagestreamtags"),
```

Comment:
> what is imagestreams/secrets?

Added a while back to support secured import.  It is a way to interact the secrets that are valid for use in a keyring for a given import attempt.